src/App.js

```diff
@@ -8,14 +8,17 @@ import Modal from './Modal'
 class App extends Component {
   constructor(){
     super()
-    this.state = movieData    
+    this.state = {movies: movieData,
+    wasClicked: []
```

Comment:
Hm - why is wasClicked an array? I'd be curious why we need this piece of state in the first place. What's yr reasoning here?